src/main/java/com/yahoo/identity/services/session/LoggedInSessionImpl.java

```diff
@@ -0,0 +1,91 @@
+package com.yahoo.identity.services.session;
+
+import com.yahoo.identity.services.account.Account;
+import com.yahoo.identity.services.account.AccountService;
+import com.yahoo.identity.services.account.AccountUpdate;
+import com.yahoo.identity.services.credential.Credential;
+import com.yahoo.identity.services.credential.CredentialImpl;
+import com.yahoo.identity.services.credential.CredentialService;
+import com.yahoo.identity.services.credential.CredentialServiceImpl;
+import com.yahoo.identity.services.storage.Storage;
+import com.yahoo.identity.services.storage.sql.SqlAccountService;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+import javax.annotation.Nonnull;
+
+public class LoggedInSessionImpl implements LoggedInSession {
```

Comment:
`LoggedInSessionImpl` can extends from SessionImpl, and implements `LoggedInSession`.
Because `LoggedInSessionImpl` extends the functionality of `SessionImpl`.

Reply:
I generally agree with the idea but there is one thing in the `SessionImpl:`sessionAccountCreate`(create with anonymous session), which shouldn't be inherited by the `LoggedInSession`. Any idea how to deal with that?

Reply:
Why you think `LoggedInSession` cannot have `sessionAccountCreate`?
A logged in user should still have the permission to create another account.

Reply:
Nice point here... Will think about the process and address that in next PR.